package.json

```diff
@@ -1,81 +1,75 @@
-  },
-  "homepage": "https://github.com/blackbaud/skyux-pages#readme",
   "peerDependencies": {
-    "@angular/common": "^9 || ^12",
-    "@angular/core": "^9 || ^12",
-    "@skyux/i18n": "^5.0.0-beta.0",
-    "@skyux/indicators": "^5.0.0-beta.0",
-    "@skyux/layout": "^5.0.0-beta.0",
-    "@skyux/router": "^5.0.0-beta.0",
-    "@skyux/theme": "^5.0.0-beta.0"
+    "@skyux/assets": "^5.0.0-beta.2",
+    "@skyux/i18n": "^5.0.0-beta.7"
   },
-  "dependencies": {},
   "devDependencies": {
-    "@angular/animations": "9.1.13",
-    "@angular/common": "9.1.13",
-    "@angular/compiler": "9.1.13",
-    "@angular/compiler-cli": "9.1.13",
-    "@angular/core": "9.1.13",
-    "@angular/forms": "9.1.13",
-    "@angular/platform-browser": "9.1.13",
-    "@angular/platform-browser-dynamic": "9.1.13",
-    "@angular/router": "9.1.13",
-    "@blackbaud/auth-client": "2.48.0",
-    "@blackbaud/skyux-lib-clipboard": "4.0.0",
-    "@blackbaud/skyux-lib-code-block": "4.0.3",
-    "@blackbaud/skyux-lib-media": "4.0.0",
-    "@blackbaud/skyux-lib-restricted-view": "4.2.1",
-    "@blackbaud/skyux-lib-stache": "4.2.2",
-    "@skyux-sdk/builder": "4.9.0",
-    "@skyux-sdk/builder-code-formatter": "4.0.0-rc.0",
-    "@skyux-sdk/builder-plugin-skyux": "4.2.0",
-    "@skyux-sdk/e2e": "4.0.2",
-    "@skyux-sdk/testing": "4.3.0",
-    "@skyux/animations": "4.0.1",
-    "@skyux/assets": "4.0.1",
-    "@skyux/config": "4.4.0",
-    "@skyux/core": "4.6.2",
-    "@skyux/docs-tools": "4.9.1",
-    "@skyux/forms": "4.20.0",
-    "@skyux/http": "4.2.0",
-    "@skyux/i18n": "4.2.1",
-    "@skyux/indicators": "4.10.0",
-    "@skyux/inline-form": "4.1.1",
-    "@skyux/layout": "4.7.1",
-    "@skyux/lists": "4.8.0",
-    "@skyux/lookup": "4.17.0",
-    "@skyux/modals": "4.5.5",
-    "@skyux/omnibar-interop": "4.0.1",
-    "@skyux/pages": "4.0.0-beta.0",
-    "@skyux/popovers": "4.5.3",
-    "@skyux/router": "4.2.2",
-    "@skyux/tabs": "4.6.7",
-    "@skyux/theme": "4.17.0",
-    "codelyzer": "5.2.2",
-    "rxjs": "6.6.7",
+    "@angular-devkit/build-angular": "~12.2.5",
+    "@angular-eslint/builder": "12.3.1",
+    "@angular-eslint/eslint-plugin": "12.3.1",
+    "@angular-eslint/eslint-plugin-template": "12.3.1",
+    "@angular-eslint/schematics": "12.4.1",
+    "@angular-eslint/template-parser": "12.3.1",
+    "@angular/cli": "~12.2.5",
+    "@angular/compiler-cli": "~12.2.5",
+    "@skyux-sdk/documentation-schematics": "^5.0.0-beta.4",
+    "@skyux-sdk/e2e": "^5.0.0-beta.1",
+    "@skyux-sdk/testing": "^5.0.0-beta.2",
+    "@skyux/assets": "^5.0.0-beta.2",
+    "@skyux/config": "^5.0.0-beta.2",
+    "@skyux/core": "^5.0.0-beta.11",
+    "@skyux/e2e-client": "^5.0.0",
+    "@skyux/forms": "^5.0.0-beta.4",
+    "@skyux/i18n": "^5.0.0-beta.7",
+    "@skyux/indicators": "^5.0.0-beta.4",
+    "@skyux/layout": "5.0.0-beta.7",
+    "@skyux/modals": "^5.0.0-beta.5",
+    "@skyux/packages": "^5.0.0-beta.92",
+    "@skyux/router": "^5.0.0-beta.2",
+    "@skyux/theme": "^5.0.0-beta.5",
```

Comment:
Some of these `devDependencies` should be placed in the `dependencies` section. Whatever `skyux eject` created should be used.